src/Oscoin/API/HTTP/Internal.hs

```diff
@@ -146,6 +154,14 @@ runApi :: Api c tx s i ()
 runApi app port hdl =
     runSpock port (mkMiddleware app hdl)
 
+-- | Like 'runApi', but it doesn't display the 'Spock is running' banner.
```

Comment:
Aww, please make this the default!

Reply:
Hehe, yes, the default banner can be annoying -- definitely 👍 to make this the default, unless @geigerzaehler or @cloudhead are really attached to the banner 😁

Reply:
If we could instead funnel it through the standard logger, that'd be great

Reply:
I suppose we could extend `runApiWithBanner` to be a bit more powerful -- namely, to be passed a `Logger` and perhaps this `IO ()` callback to be used for testing. We could even rename it to simply `runApi'` which would now be a (more complex) version of `runApi`, which could then be implemented in terms of `runApi'`. Will give it a stab 😏

Reply:
Oh, but we will still need a 3rd function for the banner/no-banner I suppose? 🤔 (which again could be implemented by `runApi'` with a boolean-of-sort to display or not the banner).

Reply:
98e8dcb6 Didn't touch `runApi` though, we can do that later in a separate PR.